### PR TITLE
(IAC-746) - Add ubuntu 20.04 support

### PR DESCRIPTION
**Pull request by david22swan**
IAC-746
pdk version: `1.17.0`

### CODE REVIEW

File: provision.yaml [hunk 1]
```diff
@@ -20,3 +23,6 @@ travis_el7:
 release_checks:
   provisioner: vmpooler
   images: ['redhat-5-x86_64', 'redhat-6-x86_64', 'redhat-7-x86_64', 'redhat-8-x86_64', 'centos-5-x86_64', 'centos-6-x86_64', 'centos-7-x86_64', 'centos-8-x86_64', 'oracle-6-x86_64', 'scientific-6-x86_64', 'scientific-7-x86_64', 'debian-8-x86_64', 'debian-9-x86_64', 'debian-10-x86_64', 'ubuntu-1404-x86_64', 'ubuntu-1604-x86_64', 'ubuntu-1804-x86_64']
+release_checks:
```
Comment (carabasdaniel) [inline provision.yaml:26]: If you want to separate the jobs for Litmus acceptance you can split the release checks for puppet 5/6 if needed, but if you are using the same machines you can keep a single group, either way you will need to clean up this provisioning file.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -6,14 +6,36 @@ on:
...
+  LitmusAcceptancePuppet6:
+    env:
+      HONEYCOMB_WRITEKEY: 7f3c63a70eecc61d635917de46bea4e6
+      HONEYCOMB_DATASET: litmus tests
+    runs-on: self-hosted
+    strategy:
+      matrix:
+        ruby_version: [2.5.x]
+        puppet_gem_version: [~> 6.0]
+        platform: [release_checks_6]
```
Comment (carabasdaniel) [inline .github/workflows/release.yml:37]: If you use the same resources for both versions you can keep only one release_checks group in your provisioning file

Comment (david22swan) [reply .github/workflows/release.yml:37]: my bad, fixed
Also rebased onto master to include abs changes

**Comment by carabasdaniel** [comment]
Looks good. Can you please remove the failing OSes from the provision lists and add a ticket for a fix on these failures ?

**Pull request merged**